contributing/release-schedule.md

```diff
@@ -87,51 +84,41 @@ To opt your browser into the AMP Dev Channel, go to [the AMP experiments page](h
+Similar to the _Dev Channel_, when you opt into the _AMP Beta Channel_ you are only affecting the AMP JS libraries in your browser; there is no way to force visitors to your site to use the _AMP Beta Channel_ version of AMP.
 
-Similar to the Dev channel, when you opt into the AMP RC Channel you are only affecting the AMP JS libraries in your browser; there is no way to force visitors to your site to use the AMP RC Channel version of AMP.
+To opt your browser into the _AMP Beta Channel_, go to [the AMP experiments page](https://cdn.ampproject.org/experiments.html) and activate the "AMP RC Channel" experiment. Please subscribe to our [low-volume announcements](https://groups.google.com/forum/#!forum/amphtml-announce) mailing list to get notified about important/breaking changes about AMP.
 
-To opt your browser into the AMP RC Channel, go to [the AMP experiments page](https://cdn.ampproject.org/experiments.html) and activate the "AMP RC Channel" experiment. Please subscribe to our [low-volume announcements](https://groups.google.com/forum/#!forum/amphtml-announce) mailing list to get notified about important/breaking changes about AMP.
+> Note: "RC" refers to the term "Release Candidate", which was the previous name for the **beta** release channel.
 
-**If you find an issue that appears to only occur in the RC Channel version of AMP**:
+**If you find an issue that appears to only occur in the _Beta Channel_ version of AMP**:
 
 - please [file an issue](https://github.com/ampproject/amphtml/issues/new) with a description of the problem
-  - include a note that the problem is new to the RC Channel build so that it can be properly prioritized
+  - include a note that the problem is new to the _Beta Channel_ build so that it can be properly prioritized
   - include a URL to a page that reproduces the problem
-- ping the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)) with the issue you filed so we can delay the push of the RC Channel version to production if needed
-
-## Release cadence
-
-We are intentionally cautious with our release cadence.
-
-In determining how often we should push new versions of AMP to everyone, we have to weigh many factors including:
-
-- stability for the millions of sites/billions of pages built using AMP
-- cache busting that might happen when we push a new version
-- the desire to get new features out quickly
-
-After considering all of these factors we have arrived at the 1-2 week push cycle. Thus far we have found this to be a reasonable compromise, but we will continue to evaluate all of these factors and may make changes in the future.
+- ping the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)) with the issue you filed so we can delay the push of the _Beta Channel_ version to production if needed
 
-## Release freezes
+## Release Freezes
```

Comment:
What is the effect on lts?

We have tried to avoid lts being part affected by our known release freezes, but it's possible there would be a release freeze for some other reason.  To avoid ambiguity we should document what happens here.

Reply:
I'd like to address this, along with more detailed explanation of the exact LTS release schedule, in a follow-up PR so we can have a more focused discussion. This thread has become massive and hard to follow. Will be creating the follow-up PRs immediately after merge!